### ISSUE TITLE
Hyperlinks break between favicon and link

### ISSUE THREAD

**Issue by hannesfrank** [remnoteio/remnote-issues#838]
Usually links break only at word delimiters (e.g. `-`) which can leave the favicon on the previous line:
![image](https://user-images.githubusercontent.com/4137110/108618318-660e0300-741d-11eb-8eae-b39beeb74dea.png)

This fixes it for me:
```css
.LinkNode {
  word-break: break-all;
}
```

A downside is that this will not break at `-` anymore, but having no break between favicon and link text seems better to me.

**Comment by drkvj** [comment]
To add on to the issue, there currently isn't a way to escape out of the hyperlink editing mode after pasting it. the only way is to press enter and leave the line altogether. 
Could we maybe have like CMD/CTRL + Enter that stops the edit mode and still stays in the line, or maybe the right arrow key stroke after the last letter get's it out?

**Comment by drkvj** [comment]
Ah, End scrolls to the bottom page for me on Mac. Or are you referring to enter?

**Comment by hannesfrank** [comment]
No, I mean the shortcut that goes to the end of a line. Maybe Cmd + -> works on Mac. There are a few more: https://apple.stackexchange.com/questions/16135/remap-home-and-end-to-beginning-and-end-of-line Have not tested this personally.

**Comment by drkvj** [comment]
Oh wow, thanks that definitely makes life easier, didn't know that CMD + -> can take us out of formating environment
